src/mqt/predictor/rl/Predictor.py

```diff
@@ -104,13 +180,23 @@ def evaluate_sample_circuit_using(self, file):
             seed_transpiler=1,
         )
         duration = time.time() - start_time
-        qiskit_o3_fid = np.round(
-            reward.expected_fidelity(transpiled_qc_qiskit, "ibm_washington"), 2
+        rew_fid = reward.expected_fidelity(transpiled_qc_qiskit, "ibm_washington")
+        rew_crit_depth = reward.crit_depth(transpiled_qc_qiskit)
+        rew_gate_ratio = reward.gate_ratio(transpiled_qc_qiskit)
+        rew_mix = reward.mix(transpiled_qc_qiskit, "ibm_washington")
+
+        return rl.Result(
+            benchmark=benchmark,
+            used_setup=used_setup,
+            time=duration,
+            fidelity=rew_fid,
+            depth=rew_crit_depth,
+            gate_ratio=rew_gate_ratio,
+            mix=rew_mix,
         )
```

Comment:
as above. this should essentially be just a single line of code.

Reply:
done